src/subprocess_tee/__init__.py

```diff
@@ -64,14 +66,24 @@ async def _stream_subprocess(args: str, **kwargs: Any) -> CompletedProcess:
 
         # Some users are reporting that default (undocumented) limit 64k is too
         # low
-        process = await asyncio.create_subprocess_shell(
-            args,
-            limit=STREAM_LIMIT,
-            stdin=kwargs.get("stdin", False),
-            stdout=asyncio.subprocess.PIPE,
-            stderr=asyncio.subprocess.PIPE,
-            **platform_settings,
-        )
+        if isinstance(args, str):
```

Comment:
i think a potentially wiser check might be to replicate what `subprocess` does, as in force people to be explicit about using "shell mode":
```
If passing a single string, either shell must be [True](https://docs.python.org/3/library/constants.html#True) (see below) or else the string must simply name the program to be executed without specifying any arguments.
```

Reply:
huh, i did _not_ know you could review PRs on repos you can't write to

Reply:
also, need to remove lines 141-147